ts/utils/stringBuilder.ts

```diff
@@ -24,6 +24,17 @@ export const formatNumberAmount = (
     format: displayCurrency ? "€ %n" : "%n"
   });
 
+export const formatNumberWithNoDigits = (
```

Comment:
Could you leave a comment and write some tests on it?

Reply:
Test added in 
6d1e5cf